modules/machine-autoscaler-crd.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * control-plane-management/applying-autoscaling.adoc
+
+[id='machine-autoscaler-crd-{context}']
+= MachineAutoscaler resource definition
+
+This MachineAutoscaler resource definition shows the parameters and sample
+values for the MachineAutoscaler.
+
+
+[source,yaml]
+----
+apiVersion: "autoscaling.openshift.io/v1alpha1"
+kind: "MachineAutoscaler"
+metadata:
+  name: "worker-us-east-1a" <1>
+  namespace: "openshift-cluster-api"
+spec:
+  minReplicas: 1 <2>
+  maxReplicas: 12 <3>
+  scaleTargetRef:
+    apiVersion: machine.openshift.io/v1alpha1
```

Comment:
Oops, this may be a type in our example upstream. This should be: `machine.openshift.io/v1beta1` I think. It's no longer alpha.